common/message/src/main/java/org/thingsboard/server/common/msg/core/BasicTelemetryUploadRequestForDepth.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright © 2016-2017 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.common.msg.core;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import lombok.ToString;
+import org.thingsboard.server.common.data.kv.KvEntry;
+import org.thingsboard.server.common.msg.session.MsgType;
+
+@ToString
+public class BasicTelemetryUploadRequestForDepth extends BasicRequest implements TelemetryUploadRequestForDepth {
```

Comment:
Does it make sense to rename it to BasicDepthTelemetryUploadRequest and Interface to DepthTelemetryUploadRequest ?